.husky/pre-push

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+# Branch names must be kebab-case
+
+npx enforce-branch-name '^([a-z][a-z0-9]*)(-[a-z0-9]+)*$'
```

Comment:
Do we need to add `enforce-branch-name` to package.json, or does it not matter as we're using `npx`?

Reply:
Yes, that's right 🙂  Because we are using `npx` there is no need to install the package 👍